policyengine_us/parameters/gov/irs/capital_gains/thresholds.yaml

```diff
@@ -0,0 +1,192 @@
+    - title: 2023 IRS data release
+      href: https://www.irs.gov/pub/irs-drop/rp-22-38.pdf#page=8
+    - title: 2022 IRS data release
+      href: https://www.irs.gov/pub/irs-drop/rp-21-45.pdf#page=8
+    - title: 2021 IRS data release
+      href: https://www.irs.gov/pub/irs-drop/rp-20-45.pdf#page=8
+    - title: 2020 IRS data release
+      href: https://www.irs.gov/pub/irs-drop/rp-19-44.pdf#page=8
+    - title: 2019 IRS data release
+      href: https://www.irs.gov/pub/irs-drop/rp-18-57.pdf#page=8
+1:
+  HEAD_OF_HOUSEHOLD:
+    values:
+      2013-01-01: 48_600
+      2014-01-01: 49_400
+      2015-01-01: 50_200
+      2016-01-01: 50_400
+      2017-01-01: 50_800
+      2018-01-01: 51_700
+      2019-01-01: 52_750
+      2020-01-01: 53_600
+      2021-01-01: 54_100
+      2022-01-01: 55_800
+      2023-01-01: 59_750
+      2024-01-01: 63_000
+      2025-01-01: 64_750
+      2026-01-01: 66_200
+  JOINT:
+    values:
+      2013-01-01: 72_500
+      2014-01-01: 73_800
+      2015-01-01: 74_900
+      2016-01-01: 75_300
+      2017-01-01: 75_900
+      2018-01-01: 77_200
+      2019-01-01: 78_750
+      2020-01-01: 80_000
+      2021-01-01: 80_800
+      2022-01-01: 83_350
+      2023-01-01: 89_250
+      2024-01-01: 94_050
+      2025-01-01: 96_700
+      2026-01-01: 98_900
+  SEPARATE:
+    values:
+      2013-01-01: 36_250
+      2014-01-01: 36_900
+      2015-01-01: 37_450
+      2016-01-01: 37_650
+      2017-01-01: 37_950
+      2018-01-01: 38_600
+      2019-01-01: 39_375
+      2020-01-01: 40_000
+      2021-01-01: 40_400
+      2022-01-01: 41_675
+      2023-01-01: 44_625
+      2024-01-01: 47_025
+      2025-01-01: 48_350
+      2026-01-01: 49_450
+  SINGLE:
+    values:
+      2013-01-01: 36_250
+      2014-01-01: 36_900
+      2015-01-01: 37_450
+      2016-01-01: 37_650
+      2017-01-01: 37_950
+      2018-01-01: 38_600
+      2019-01-01: 39_375
+      2020-01-01: 40_000
+      2021-01-01: 40_400
+      2022-01-01: 41_675
+      2023-01-01: 44_625
+      2024-01-01: 47_025
+      2025-01-01: 48_350
+      2026-01-01: 49_450
+  SURVIVING_SPOUSE:
+    values:
+      2013-01-01: 72_500
+      2014-01-01: 73_800
+      2015-01-01: 74_900
+      2016-01-01: 75_300
+      2017-01-01: 75_900
+      2018-01-01: 77_200
+      2019-01-01: 78_750
+      2020-01-01: 80_000
+      2021-01-01: 80_800
+      2022-01-01: 83_350
+      2023-01-01: 89_250
+      2024-01-01: 94_050
+      2025-01-01: 96_700
+      2026-01-01: 98_900
+2:
+  HEAD_OF_HOUSEHOLD:
+    values:
+      2013-01-01: 425_000
+      2014-01-01: 432_200
+      2015-01-01: 439_000
+      2016-01-01: 441_000
+      2017-01-01: 444_550
+      2018-01-01: 452_400
+      2019-01-01: 461_700
+      2020-01-01: 469_050
+      2021-01-01: 473_750
+      2022-01-01: 488_500
+      2023-01-01: 523_050
+      2024-01-01: 551_350
+      2025-01-01: 566_700
+      2026-01-01: 579_600
+  JOINT:
+    values:
+      2013-01-01: 450_000
```

Comment:
Nit but we don't need the `values` for each of the breakdown parameters if there is no metadata